andela/travel_tool_front
# #159725777 Implement request details modal

#### What does this PR do?
This PR implementS an expanded view of a travel request so that users can perform necessary actions or view updates on the request.

#### Description of Task to be completed?
- Create Request modal
- Implement  Request modal to be mobile responsive
- Create Text editor for comment box 
- Implement the different states of the WYISWYG editor 
- Write test  for the Request modal component

#### How should this be manually tested?
- Clone the repository
- Run `make start` docker build command on the terminal to install all dependencies and run the App in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:3000` and Login with your Andela email account into the App.
- On the Requests page, click on the request Id from the list of request displayed on the request table, you should be able to see the Request details Modal displayed.


#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#159725777](https://www.pivotaltracker.com/story/show/159725777)


#### Screenshots (if appropriate)

<img width="1280" alt="screen shot 2018-08-26 at 20 48 15" src="https://user-images.githubusercontent.com/12033494/44631217-b6ea3100-a971-11e8-872d-153f8795a2c1.png">



#### Questions:
None
